code/lib/preview-api/src/modules/store/csf/portable-stories.ts

```diff
@@ -54,18 +106,17 @@ export function composeStory<TRenderer extends Renderer = Renderer, TArgs extend
     storyAnnotations.storyName ||
     storyAnnotations.story?.name ||
     storyAnnotations.name ||
-    'unknown';
+    'Unnamed Story';
 
   const normalizedStory = normalizeStory<TRenderer>(
     storyName,
     storyAnnotations,
     normalizedComponentAnnotations
   );
 
-  const normalizedProjectAnnotations = normalizeProjectAnnotations<TRenderer>({
-    ...projectAnnotations,
-    ...defaultConfig,
-  });
+  const normalizedProjectAnnotations = normalizeProjectAnnotations<TRenderer>(
+    mergeAnnotations(projectAnnotations, defaultConfig)
```

Comment:
shouldn't this be the other way around? with `projectAnnotations` overriding `defaultConfig`?